src/uct/ib/ud/base/ud_ep.c

```diff
@@ -786,13 +806,7 @@ static void uct_ud_ep_rx_ctl(uct_ud_iface_t *iface, uct_ud_ep_t *ep,
 
     /* Discard duplicate CREP */
     if (UCT_UD_PSN_COMPARE(neth->psn, <, ep->rx.ooo_pkts.head_sn)) {
-        ucs_assertv_always(ep->flags & UCT_UD_EP_FLAG_CREP_RCVD,
-                           "iface=%p ep=%p conn_sn=%d ep_id=%d, dest_ep_id=%d"
-                           " rx_psn=%u neth_psn=%u ep_flags=0x%x ctl_ops=0x%x"
-                           " rx_creq_count=%d",
-                           iface, ep, ep->conn_sn, ep->ep_id, ep->dest_ep_id,
-                           ep->rx.ooo_pkts.head_sn, neth->psn, ep->flags,
-                           ep->tx.pending.ops, ep->rx_creq_count);
+        uct_ud_ep_rx_ctl_drop_packet(ep, neth);
```

Comment:
what if ep->flags & UCT_UD_EP_FLAG_CREP_RCVD is false, but (ep->flags & UCT_UD_EP_FLAG_CREQ_RCVD) && (ep->rx_creq_count > 1)) is true?
in such case, assertion used to fail, but now it wouldn't fail

Reply:
fixed